histoqc/BaseImage.py

```diff
@@ -70,7 +88,7 @@ def __init__(self, fname, fname_outdir, params):
         mask_statistics_types = ["relative2mask", "absolute", "relative2image"]
         if (self["mask_statistics"] not in mask_statistics_types):
             logging.error(
-                f"mask_statistic type '{self['mask_statistics']}' is not one of the 3 supported options relative2mask, absolute, relative2image!")
+                f"mask_statistic type \"{self['mask_statistics']}\" is not one of the 3 supported options relative2mask, absolute, relative2image!")
```

Comment:
out of curiosity, did you do this change manually or was it part of an IDE formatting? is there a reason for it or personal preference?

Reply:
i agree, but looking at the other code i think the single ones are used _in_ a print statement and a double quote is used _around_ the print statement. in this way we don't have to do escape the quotes (which i always fail to do and thus create more errors for myself : )  )